src/libs/vtkh/filters/SampleGrid.cpp

```diff
@@ -0,0 +1,89 @@
+
+#include <vtkh/filters/SampleGrid.hpp>
+#include <vtkh/Error.hpp>
+
+#include <vtkh/vtkm_filters/vtkmProbe.hpp>
+#include <limits>
+
+namespace vtkh
+{
+
+SampleGrid::SampleGrid()
+	: m_invalid_value(std::numeric_limits<double>::min())
+{
+
+}
+
+SampleGrid::~SampleGrid()
+{
+
+}
+
+void
+SampleGrid::PreExecute()
+{
+  Filter::PreExecute();
+}
+
+void
+SampleGrid::DoExecute()
+{
+  this->m_output = new DataSet();
+
+  const int num_domains = this->m_input->GetNumberOfDomains();
+
+  for(int i = 0; i < num_domains; ++i)
+  {
```

Comment:
I think we need more logic here to support the distributed memory case. 

If all ranks are doing the resample to the same grid, we would have do a reduction to compose the results on each task. 

Also, we may want to distribute the output grid across MPI tasks as well for high res resampling.

Reply:
I should add a parallel test

Reply:
yes - I think so.  Let's chat about the cases high bandwidth. 

To test this you can use the 2D spiral mulit-domain blueprint example and dish out domains to different mpi tasks.